lib/build.js

```diff
@@ -145,10 +145,23 @@ class BuildModel extends BaseModel {
             },
 
             (response, next) => {
-                // Create github status
+                // Get the admin of the pipeline
+                const pipelineModel = new PipelineModel(this.datastore);
```

Comment:
There is already a spot where the pipelineModel is instantiated, can we refactor that to happen only once?